app/managers.py

```diff
@@ -0,0 +1,55 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(
+            "C:\\Users\\vlada\\py-actor-manager\\identifier.sqlite"
+        )
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT "

```

Comment:
It is common to use triple quotes for multilne queries
Also check sql format so it follows the standards. This resource will help https://sqlformat.org/